pixart.js

```diff
@@ -1 +1,34 @@
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
+var form = document.querySelector("#form");
+var body = document.querySelector("body");
```

Comment:
just be a little careful about naming.. since you use things like ```document.body```, you dont want to confuse it with the variable ```body``` that you've created